index.html

```diff
@@ -10,10 +10,8 @@
 <body>
     <div class="main-container">
         <header class="mobile-header">
-            <h1 id="name">Hans Paul Zizold</h1>
-            <button class="menu">
-                <img src="images/menu.png" alt="Menu">
-            </button>
+            <p id="name">Hans Paul Zizold</p>
+            <img src="images/menu.png" alt="Menu">
```

Comment:
As suggested by the previous  reviewer it's better here to use an `a` tag instead, as it will make your code look more
professional . Kindly fix it 😃✔